src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -153,8 +168,9 @@ export class LdPagination {
     this.visibleItemsInSlider = this.offset * 2 + 1
     const maxVisibleItems = this.sticky * 2 + this.visibleItemsInSlider
     this.maxSliderColumns = this.visibleItemsInSlider + 2
-    this.renderSticky = this.sticky > 0
-    this.renderMoreIndicators = this.length > maxVisibleItems + 2
+    this.renderSticky = this.sticky > 0 && this.mode !== 'dots'
```

Comment:
Please add `@Watch('mode')` decorator on the `componenWillLoad` lifecycle method to re-evaluate this on `mode` prop changes.